components/org.wso2.carbon.identity.api.server.endpoint.configuration.management/org.wso2.carbon.identity.api.server.endpoint.configuration.management.common/src/main/java/org/wso2/carbon/identity/api/server/endpoint/configuration/management/common/ContextLoader.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.server.endpoint.configuration.management.common;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.base.MultitenantConstants;
+import org.wso2.carbon.context.PrivilegedCarbonContext;
+import org.wso2.carbon.identity.api.server.endpoint.configuration.management.common.error.APIError;
+import org.wso2.carbon.identity.api.server.endpoint.configuration.management.common.error.ErrorResponse;
+import org.wso2.carbon.identity.core.ServiceURLBuilder;
+import org.wso2.carbon.identity.core.URLBuilderException;
+import org.wso2.carbon.identity.core.util.IdentityTenantUtil;
+import org.wso2.carbon.identity.core.util.IdentityUtil;
+
+import java.net.URI;
+
+import javax.ws.rs.core.Response;
+
+import static org.wso2.carbon.identity.api.server.endpoint.configuration.management.common.EndpointConfigurationManagementConstants.SERVER_API_PATH_COMPONENT;
+import static org.wso2.carbon.identity.api.server.endpoint.configuration.management.common.EndpointConfigurationManagementConstants.TENANT_CONTEXT_PATH_COMPONENT;
+import static org.wso2.carbon.identity.api.server.endpoint.configuration.management.common.EndpointConfigurationManagementConstants.TENANT_NAME_FROM_CONTEXT;
+import static org.wso2.carbon.identity.application.common.util.IdentityApplicationConstants.Error.UNEXPECTED_SERVER_ERROR;
+
+/**
+ * Load information from context.
+ */
+public class ContextLoader {
```

Comment:
Context loader is already available in [here](https://github.com/wso2/identity-api-server/blob/v1.0.237/components/org.wso2.carbon.identity.api.server.common/src/main/java/org/wso2/carbon/identity/api/server/common/ContextLoader.java). Do we have any special requirement to have a new one?

Reply:
resolved